vermouth/tests/test_ff_files.py

```diff
@@ -0,0 +1,1019 @@
+# Copyright 2018 University of Groningen
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Test that force field files are properly read.
+"""
+
+import textwrap
+import pytest
+import numpy as np
+import vermouth.ffinput
+import vermouth.forcefield
+import vermouth.molecule
+
+
+class TestBlock:
+    @staticmethod
+    def test_moleculetype():
+        lines = """
+        [ moleculetype ]
+        GLY  3
+        """
+        lines = textwrap.dedent(lines).splitlines()
+        ff = vermouth.forcefield.ForceField(name='test_ff')
+        vermouth.ffinput.read_ff(lines, ff)
+        assert 'GLY' in ff.blocks
+        assert ff.blocks['GLY'].nrexcl == 3
+
+    @staticmethod
+    def test_atoms():
+        lines = """
+        [ moleculetype ]
+        GLY 1
+
+        [ atoms ]
+          1 P4 1 ALA BB 1
+        2 P3\t1\tALA   SC1 2 -3 7
```

Comment:
indent change?

Reply:
This is on purpose, the point is to make sure that trailing spaces at the beginning of the line do not cause an error.